helm/spark-yotpo/values.yaml

```
@@ -0,0 +1,53 @@
+version: 0.2
+sparkVersion: 2.4.5
+image: metorikku/k8s-spark-operator
+jar: file:///target/metorikku.jar
+mainClass: com.yotpo.metorikku.Metorikku
+fileDependencies: []
+arguments:
+  - -c
```

```suggestion
  - "-c"
```
Not sure it's required